tools/lib/provision.py

```diff
@@ -197,8 +199,12 @@
         *VENV_DEPENDENCIES,
     ]
 elif "debian" in os_families():
+    DEBIAN_DEPENDECIES = UBUNTU_COMMON_APT_DEPENDENCIES
+    if distro_info["VERSION_CODENAME"] == "bullseye":
```

Comment:
Let's add a comment documenting this hack.

Reply:
Okay.